light_peft_patch.py

```
@@ -126,59 +134,85 @@ def load_4bit_causal_lm(
+        model.gradient_checkpointing_enable(
+            gradient_checkpointing_kwargs={"use_reentrant": False}
+        )
     except TypeError:
         model.gradient_checkpointing_enable()
 
     # Ensure grads can flow into embeddings when base weights are frozen
     if hasattr(model, "enable_input_require_grads"):
         model.enable_input_require_grads()
     else:
+
         def _req_grad_hook(module, inputs, output):
```

Ruff (0.13.3)

Unused function argument: `module`

(ARG001)